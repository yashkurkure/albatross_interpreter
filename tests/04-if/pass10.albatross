if (226+179|64+60&98|38-83*106+181+82&139-140+122&124*16&58|218-77) {if (64&12&67*144+110+7|211+164|22|108|159|142-186+43|198|147|56) {if (119|183-222&150-30|226|188*243-5&44+7*100|243+26|34|124&255) {if (56) {if (49+217-144&32-132|217+42&95&209|146|115*8*98|140&147|125|246+27&65) {printint( 179-82|34+121|113|7&77+112&37&95|113+96|86-38*246*221-128-107-76|183|36|53|41&52&9*94|251*174&136);}}}}}
if (9+70|157|183-77-87+128*40&72+30*140-243*134+149*63|157&177+53|91&229|67+191*99-178+17) {if (33*58-33+231-241|218|147&32+126|63+150|72*252&137-180-42) {if (146*72|97|133|45-137*100-242-5&86*232+20-42+235+202-241&67|96|112*125*5|133|56) {if (14-110|97|51|20|121|201&180-103*158+196*103-54&53&182*89|183*190|110+155|34) {if (164-224*163&189|52&49-5&183+210+202|26*178&88+101-174|84|44*150-69+34|50|0) {printint( 240);}}}}}
if (247+87-67|216&11&41*199|185-157*140*208-85|62-162*174*212|241*148&7&232+9+91) {if (2&2|216*44|121*122*55-20-110*89+143*120|112|186+215*1+101|54-191*230-52*122|64|124*194&200*223|152-70+197) {if (79&72) {if (21*14+227|199|102|145&21+152*198-202|130*11|106&227|130-217+24|112-154|86&131|120*210&209-100) {if (59) {printint( 233-164&70|77|48&10&38&7|158*66|93+22|35|1|184*58*1+84*9|16*226);}}}}}
if (54*133-15+158|200+118|70-68-135+73|177&38-4|98&148-32+106+47-201*168*24|101) {if (140|26*146|171-108*86&21|86|8|75|225*227&48-211+110-233*159+231*227|158) {if (18+204+247&28|99*165|82&185|197|104) {if (158|104&195|54&31&129+233&9*176-96-116*176+180&158+65|135|85|237-235|118|64&111|80+224-2) {if (236*28|148|223*139-62|160+81*94&82|171-71|22+230+179|176+202|73*161*103&237) {printint( 184|238|140&47+250-242-215-136+4|224&37&34*29|128-86-87|6|244+105);}}}}}
if (89|89-77&143-15|49+17&179|238|129-199+62&130|91|51-77*57-25-210|236+172*243|129|200|216*84) {if (210&60&134|250*245+97+120&31+31+62|221|8|211-37-64+43*176+199|28-65|99|25+124*238*123&169-67|105) {if (83+137&116|240|223&87-174|96*129-82|85|162*42|214) {if (29) {if (103-232-84-62-157+101&58+9|222+25+133-127+238+177+61*79|70&221|85-253&47*21*81&214|231|56&122|2-88&51|188) {printint( 104|149+186*133*153|178+206|82-124+145-133|207|211*96+101+128*4*227-248*83+9|125|228);}}}}}
if (248|69|133&253&41|198|229+13-218-191*0&193-193+132) {if (112|233-60+194+101*162+99|124+64*127-16+131-117|31&19&19*2|134&91-204-122-217*33+16|21-211&49-247) {if (14-92+247|61|24+98|249|25*214*36|181*49|96|85|153|95|205*233|181-195*238-234-17&41+97&64&242-103*220&117&132) {if (192|38*158+206-213|234|9|116&206-236|208|117-235-164|49+127&213-33+116+77|164&18&126) {if (124|232-153-55*201*237|238|125|232-106+239*245+80+214+204|101*147|236-92) {printint( 51|110*50-52*25&55-154&123*179|3*162|11|104*79|153|248&187|23|132|220+169|69|115|253+215-148&220|204);}}}}}
if (144+217|245-198+35&12+88+142-119*232|131+76*255-242+196+59|65-24*196|203) {if (233&39|236|72|210|29|199&150|52+188-54&192*30|221+92|78|80|189*76+249-28+30+221&81+31|72) {if (37|25*63|99&21|214|123|95&233&50+225+27*46) {if (155) {if (54+145|43+152-133*237&179&95|156) {printint( 75);}}}}}
if (144*107|79-35*98&172|210|240|57-27) {if (254|184|134+219*8-202-96|231+225-224|75|125+71|149*11|192-218-160*135|93+57|117+200&34) {if (53) {if (108*237|196|8|232&116+27*21&170|9-131*86&215+79&163-62|69-245) {if (203-66*105+14&75|99-85+74|96+13-103&170-128+138-124-171|22+185|233|173&90|71+119|73*26*201|246) {printint( 57|212-120|146|186|208&100-39|81&188+13-209+234|71+13+234&121|179-51+2|191+177|183|1-97);}}}}}
if (232*83+37-33|203*38|11*251&141|234&102|54*74|215&95|56+223|143+149*37|235|40&12+43|109-65-17|126|239-59+246|87) {if (101|33|241*110|107+101*149+131+229*238-246*122+55&83+116*175&106&140&232) {if (224+96*81-29+38&85*80*138|200+200|11|210*132&88+63&206&170|45|52+118+238) {if (24) {if (107*172|33+105&125|80&242*18+60-16*58*168*81-203|249|190&8*243+38|33*88|192|7-89-222) {printint( 142|73-181|8*61-165|26&57|139+190-10+34*255|248|209*36-200*128|200+3+205*103*100*5-84+226&245+105-120);}}}}}
if (142|215|90+49-47|135*242|79-125&114+83&62*22*164&29|52*183|109|26|220*64*179|55|85|221*117|223-192) {if (80|196|53|222|169|136|236-59|163+4-53|3+17|212*24*98|240|200+207|166-93|55+189|197|116|159|54|150|251|143+152) {if (205|106|155|80|126*34-171|24&56+120) {if (180+198&139-10|133|36-146|89*46|201&123-66&22*132+169|140|236-96+250-182&68*244*215*47|140) {if (6|206-139*125|47|42-48|210|118+3|254|162*36) {printint( 248-179+231&67|67&253|105&73|249-6&14+247-148*111*23+220|139|105|178+128+94-117+180|49-88*2);}}}}}
exit(0);
