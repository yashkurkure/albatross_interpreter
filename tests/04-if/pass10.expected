-206592124012031-1-11279975-1-1-17
